src/config.c

```diff
@@ -54,5 +54,21 @@ int ReadConfig(RedisModuleString **argv, int argc, const char **err) {
     RMUtil_ParseArgsAfter("FRISOINI", argv, argc, "c", &RSGlobalConfig.frisoIni);
   }
 
+  int onTmoIndex = RMUtil_ArgIndex("ON_TIMEOUT", argv, argc);
```

Comment:
you do know that there's a util for that, right? RMUtil_ParseArgAfter or something.

Reply:
it would just simplify this code and have the same effect.